packages/client/hmi-client/src/types/common.ts

```diff
@@ -200,6 +200,12 @@ export interface CompareModelsResponseType {
 	response: string;
 }
 
+export enum ItemStatusType {
+	Completed = 'Completed',
+	Failed = 'Failed',
+	Running = 'Running'
+}
+
```

Comment:
I mean using the notification status type `Types.ts/ProgressState`

Reply:
`ProgressState` doesn't have the value `Completed` it has `Complete` no `d`.  I would have to add that value to the enum, that feel a off since over could end up using the wrong one

Reply:
That is coming from `item.status`